From 0e22ac8742a838d7dec8ccb4e2fe577fdffab828 Mon Sep 17 00:00:00 2001
From: dhruv <workbydr247@gmail.com>
Date: Tue, 23 Apr 2024 15:17:55 +0530
Subject: [PATCH] check for patch

---
 lib/screens/home_screen.dart | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/screens/home_screen.dart b/lib/screens/home_screen.dart
index 1f6cabb..e16a5df 100644
--- a/lib/screens/home_screen.dart
+++ b/lib/screens/home_screen.dart
@@ -61,7 +61,7 @@ class _HomeScreenState extends State<HomeScreen> {
             child: Container(
               padding: const EdgeInsets.all(16.0),
               child: const Text(
-                'Header Section',
+                'HeaderSection',
                 style: TextStyle(fontSize: 24.0, fontWeight: FontWeight.bold),
               ),
             ),
-- 
2.39.1.windows.1

