From 17e1c74e1af7f76334f1ffd2549401efaedb8909 Mon Sep 17 00:00:00 2001
From: dhruv <workbydr247@gmail.com>
Date: Tue, 23 Apr 2024 13:09:58 +0530
Subject: [PATCH 2/3] check for conflicts

---
 lib/screens/home_screen.dart | 1 +
 1 file changed, 1 insertion(+)

diff --git a/lib/screens/home_screen.dart b/lib/screens/home_screen.dart
index 55ff7e7..b59439c 100644
--- a/lib/screens/home_screen.dart
+++ b/lib/screens/home_screen.dart
@@ -72,6 +72,7 @@ class _HomeScreenState extends State<HomeScreen> {
               (context, index) {
                 return ListTile(
                   title: Text(items[index].toString()),
+                  leading: Text(items[index].toString()),
                 );
               },
             ),
-- 
2.39.1.windows.1

